src/entities/catalogs/services/get-catalogs-by-user.ts

```
@@ -12,20 +10,21 @@ import { db } from "~/shared/lib/firebase/config";
 export async function getCatalogByUser(userId: string) {
   let userCatalogsData: any[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'userCatalogsData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)